tests/Avalonia.RenderTests/Shapes/PathTests.cs

```diff
@@ -353,7 +353,47 @@ public async Task Path_With_PenLineCap()
             await RenderToFile(target);
             CompareImages();
         }
+        
+        [Fact]
+        public async Task PathSegment_Triggers_Invalidation_On_Property_Change()
```

Comment:
i think a better place for the test is `GeometryTests `(or dedicated `PathGeometryTests`) in `Avalonia.Visuals.UnitTests` as this is not related to rendering